config/application.rb

```diff
@@ -10,6 +10,7 @@ module UltimateBlogProject
   class Application < Rails::Application
     # Initialize configuration defaults for originally generated Rails version.
     config.load_defaults 6.0
+    config.session_store :cookie_store, expire_after: 2.hours
```

Comment:
This conflicts with the 30 minutes elsewhere. Which one takes effect?

When you have this sort of 'mostly constant' value, it's good practice to use a `const` if you can, otherwise, pop the value in a setting and use the setting everywhere. That prevents diverging values across the code base